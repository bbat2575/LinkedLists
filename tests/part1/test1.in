NEW 4
1
2.2
E
Four
VIEW 0
TYPE 0
NEW 4
One
$
3e0
4
VIEW 1
TYPE 1
VIEW ALL
REMOVE 0
VIEW ALL
REMOVE 1
VIEW ALL
