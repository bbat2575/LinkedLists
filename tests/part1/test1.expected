
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1 -> 2.20 -> E -> Four

> 1 -> 2.20 -> E -> Four

> int -> float -> char -> string

> List 1: One -> $ -> 3.00 -> 4

> One -> $ -> 3.00 -> 4

> string -> char -> float -> int

> Number of lists: 2
List 0
List 1

> List 0 has been removed.

Number of lists: 1
List 1

> Number of lists: 1
List 1

> List 1 has been removed.

Number of lists: 0

> Number of lists: 0

> 
